Sources/MeiliSearch/Request.swift

```diff
@@ -28,6 +36,30 @@ public final class Request {
     self.session = config.session ?? URLSession.shared
   }
 
+  private func request(
+    _ httpMethod: HTTPMethod,
+    _ url: URL,
+    _ data: Data?,
+    _ headers: [String: String] = [:]
+  ) -> URLRequest {
+    var request = URLRequest(url: url)
+    request.httpMethod = httpMethod.rawValue
+    if httpMethod != .get {
+      request.httpBody = data
+      request.setValue("application/json", forHTTPHeaderField: "Content-Type")
+      request.setValue("application/json; charset=utf-8", forHTTPHeaderField: "Accept")
+    }
+    request.setValue(PackageVersion.qualifiedVersion(), forHTTPHeaderField: "User-Agent")
+    headers.forEach { key, value in
+      request.addValue(value, forHTTPHeaderField: key)
+    }
+    if let apiKey: String = config.apiKey {
+      let bearer = "Bearer \(apiKey)"
+      request.addValue(bearer, forHTTPHeaderField: "Authorization")
```

Comment:
I was wondering the difference between [addValue](https://developer.apple.com/documentation/foundation/urlrequest/2011522-addvalue) and [setValue](https://developer.apple.com/documentation/foundation/urlrequest/2011447-setvalue). It seems that addvalue appends the value of a header. 

In the case of authorization, I don't think we want to append. 
```suggestion
      request.setValue(bearer, forHTTPHeaderField: "Authorization")
```